shared/store/wallet.ts

```diff
@@ -112,7 +112,9 @@ export const fetchPoolPairs = createAsyncThunk(
   'wallet/fetchPoolPairs',
   async ({ size = 200, client }: { size?: number, client: WhaleApiClient }): Promise<DexItem[]> => {
     const pairs = await client.poolpairs.list(size)
-    return pairs.map(data => ({ type: 'available', data }))
+    return pairs
+      .filter(pair => !pair.displaySymbol.includes('/v1'))
```

Comment:
Use `symbol`. I'm afraid historical tokens might not have all attributes (that a normal token have). But symbol will always be there.

Reply:
Also can you add a comment here, just so anyone knows the reason behind it